src/test/scala/org/hammerlab/guacamole/pileup/PileupSuite.scala

```diff
@@ -89,9 +86,6 @@ class PileupSuite extends GuacFunSuite with TableDrivenPropertyChecks {
       TestUtil.makeRead("TCGACCCTCGA", "4M3I4M", 1, "chr1", Some(Seq(10, 15, 20, 25, 5, 5, 5, 10, 15, 20, 25)))
     )
 
-    val noPileup = Pileup(reads, "chr1", 0, reference.getContig("chr1")).elements
-    noPileup.size should be(0)
-
```

Comment:
heh, looks like [we already test that in `PileupSuite`](https://github.com/hammerlab/guacamole/blob/14a25a141e94e367a47fb1cd1ee18f112b43a523/src/test/scala/org/hammerlab/guacamole/pileup/PileupSuite.scala#L252-L273).